api/Parser.cpp

```diff
@@ -517,6 +517,32 @@ std::vector<ConferenceEndpoint> parseConferenceEndpoints(const nlohmann::json& r
     return endpoints;
 }
 
+ConferenceEndpointExtendedInfo parseEndpointExtendedInfo(const nlohmann::json& data)
+{
+    ConferenceEndpointExtendedInfo endpoint;
+    endpoint.basicEndpointInfo = parseConferenceEndpoint(data);
+    nlohmann::json iceSelectedTuple = nlohmann::json::object();
+
+    setIfExistsOrThrow<>(iceSelectedTuple, data, "iceSelectedTuple");
```

Comment:
Isn't it selected "candidate"?

Reply:
No, it's a so-called 5-tuple {ip,port,ip,port,protocol} selected by ICE. Network-related essential info from both selected local and remote candidates.